src/main/java/com/example/rets_api/entity/PhoneEntity.java

```diff
@@ -20,22 +18,22 @@ public class PhoneEntity {
     private String primaryPhone;
     private String alternatePhone;
 
-    @OneToOne(mappedBy = "agentPhone")
+    @OneToOne(mappedBy = "agentPhone", cascade = CascadeType.ALL)
```

Comment:
I added the 'cascade' in all fields to resolve an exception when I ran the tests.